manifests/charts/istio-cni/values.yaml

```diff
@@ -72,10 +72,6 @@ cni:
       cpu: 100m
       memory: 100Mi
 
-  # Experimental taint controller for further race condition mitigation
```

Comment:
do we also need to remove https://github.com/istio/istio/blob/bc2ce8498e596cc2271f2e9a08a2632a51e0514d/operator/pkg/apis/istio/v1alpha1/values_types.proto#L79 ?